example.html

```diff
@@ -0,0 +1,345 @@
+<!DOCTYPE html>
+<html>
+
+  <head>
+    <title>Baseline grid example</title>
+    <link rel="stylesheet" href="css/example.css" />
+  </head>
+
+  <body>
+    <section>
+      <h1>Large</h1>
```

Comment:
This isn't what I had in mind. I'd like to suggest you create 3 examples (instead of 1 example.html with 3 different sized grids):
- example-15-columns-grid.html
- example-12-columns-grid.html
- example-4-columns-grid.html

... because that's exactly what's out there in real life sites!